src/components/Media.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { useLocation } from "react-router-dom";
+import { CardMedia} from '@material-ui/core';
+import ReactPlayer from 'react-player'
+import Popup from 'reactjs-popup'
+
+function Media(props){
+    const location = useLocation()
+
+    const checkLocation = () => location.pathname === '/' || location.pathname.length > 12
```

Comment:
better give it a more proper name, because it's returning a boolean and it's not clear what exactly are you checking here